software/acpid.txt

```diff
@@ -0,0 +1,155 @@
+ACPID
+________________________________________________________________________________
+
+Acpid is a daemon that executes certain actions whenever ACPI events are
+recieved. Depending on your hardware and kernel configuration, these events
+include closing a laptop lid, connecting to an AC power adapter, pressing
+certain buttons and more.
+
+
+Index
+________________________________________________________________________________
+
+- Usage                                                                    [0.0]
+    - Kernel Configuration                                                 [0.1]
+
+- Busybox acpid                                                            [1.0]
+
+- Acpid2                                                                   [2.0]
+    - Finding Events                                                       [2.1]
+    - Handling Events                                                      [2.2]
+
+
+[0.0] Usage
+________________________________________________________________________________
+
+KISS Linux offers two options for acpid management: busybox acpid, which is
+installed by default, and acpid2, which can be installed from the acpid
+package. For both acpid implimentations, you will need to enable a few
+ACPI-related kernel options and optionally the runit service to manage it.
+
+To enable the acpid runit service run:
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ ln -s /etc/sv/acpid/ /var/service/                                       |
+|   $ sv up acpid                                                              |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+Whenever changes are made, restart acpid for them to take effect.
+
++--------------------------------------------------------------------------+
+|                                                                          |
+|   $ sv restart acpid                                                     |
+|                                                                          |
++--------------------------------------------------------------------------+
+
+
+    [0.1] Kernel Configuration
+    ____________________________________________________________________________
+
+    For now, check the Gentoo Wiki for details. [0] The important information
+    will be added to this page soon.
```

Comment:
This would be best as `[1.0]` (its own section).

Reply:
Feel free to skip over the "optional acpi drivers" section of the Gentoo Wiki. Just include the mandatory config option and feel free to refer back to Gentoo for the optional stuff.